diff -uprN apache-storm-0.9.1-incubating-vanilla/bin/storm apache-storm-0.9.1-incubating/bin/storm
--- apache-storm-0.9.1-incubating-vanilla/bin/storm	2014-02-03 23:30:11.000000000 +0100
+++ apache-storm-0.9.1-incubating/bin/storm	2014-03-20 12:40:17.221361875 +0100
@@ -38,9 +38,10 @@ if sys.platform == "cygwin":
 else:
     normclasspath = identity
 
-STORM_DIR = "/".join(os.path.realpath( __file__ ).split("/")[:-2])
-USER_CONF_DIR = os.path.expanduser("~/.storm")
-CLUSTER_CONF_DIR = STORM_DIR + "/conf"
+STORM_DIR=os.getenv("STORM_DIR")
+USER_CONF_DIR=os.getenv("USER_CONF_DIR")
+CLUSTER_CONF_DIR=os.getenv("CLUSTER_CONF_DIR")
+
 if (not os.path.isfile(USER_CONF_DIR + "/storm.yaml")):
     USER_CONF_DIR = CLUSTER_CONF_DIR
 CONFIG_OPTS = []
