diff -uprN logstash-1.4.2-vanilla/bin/logstash logstash-1.4.2/bin/logstash
--- logstash-1.4.2-vanilla/bin/logstash	2014-06-22 03:14:48.000000000 +0200
+++ logstash-1.4.2/bin/logstash	2014-09-11 16:57:39.626544440 +0200
@@ -19,7 +19,7 @@
 LS_HEAP_SIZE="${LS_HEAP_SIZE:=500m}"
 
 unset CDPATH
-basedir=$(cd `dirname $0`/..; pwd)
+basedir=/usr/lib/logstash
 . "${basedir}/bin/logstash.lib.sh"
 
 setup
diff -uprN logstash-1.4.2-vanilla/bin/logstash.lib.sh logstash-1.4.2/bin/logstash.lib.sh
--- logstash-1.4.2-vanilla/bin/logstash.lib.sh	2014-06-22 03:14:48.000000000 +0200
+++ logstash-1.4.2/bin/logstash.lib.sh	2014-09-11 12:18:15.051963038 +0200
@@ -1,4 +1,4 @@
-basedir=$(cd `dirname $0`/..; pwd)
+basedir=/usr/lib/logstash
 
 setup_ruby() {
   export RUBYLIB="${basedir}/lib"
@@ -96,4 +96,4 @@ install_deps() {
   else
     echo "Cannot install dependencies; missing logstash.gemspec. This 'deps' command only works from a logstash git clone."
   fi
-}
\ No newline at end of file
+}
