diff -uprN spark-1.1.0-vanilla/bin/compute-classpath.sh spark-1.1.0/bin/compute-classpath.sh
--- spark-1.1.0-vanilla/bin/compute-classpath.sh	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/bin/compute-classpath.sh	2014-10-13 13:00:10.538168071 +0200
@@ -25,7 +25,7 @@ SCALA_VERSION=2.10
 # Figure out where Spark is installed
 FWDIR="$(cd `dirname $0`/..; pwd)"
 
-. $FWDIR/bin/load-spark-env.sh
+. /etc/spark/conf/load-spark-env.sh
 
 # Build up classpath
 CLASSPATH="$SPARK_CLASSPATH:$SPARK_SUBMIT_CLASSPATH:$FWDIR/conf"
diff -uprN spark-1.1.0-vanilla/bin/pyspark spark-1.1.0/bin/pyspark
--- spark-1.1.0-vanilla/bin/pyspark	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/bin/pyspark	2014-10-13 13:00:35.631886950 +0200
@@ -48,7 +48,7 @@ if [ ! -f "$FWDIR/RELEASE" ]; then
   fi
 fi
 
-. $FWDIR/bin/load-spark-env.sh
+. /etc/spark/conf/load-spark-env.sh
 
 # Figure out which Python executable to use
 if [[ -z "$PYSPARK_PYTHON" ]]; then
diff -uprN spark-1.1.0-vanilla/bin/spark-class spark-1.1.0/bin/spark-class
--- spark-1.1.0-vanilla/bin/spark-class	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/bin/spark-class	2014-10-13 13:01:59.755944659 +0200
@@ -32,7 +32,7 @@ FWDIR="$(cd `dirname $0`/..; pwd)"
 # Export this as SPARK_HOME
 export SPARK_HOME="$FWDIR"
 
-. $FWDIR/bin/load-spark-env.sh
+. /etc/spark/conf/load-spark-env.sh
 
 if [ -z "$1" ]; then
   echo "Usage: spark-class <class> [<args>]" 1>&2
diff -uprN spark-1.1.0-vanilla/sbin/slaves.sh spark-1.1.0/sbin/slaves.sh
--- spark-1.1.0-vanilla/sbin/slaves.sh	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/sbin/slaves.sh	2014-10-13 12:47:52.044450528 +0200
@@ -63,7 +63,7 @@ then
   shift
 fi
 
-. "$SPARK_PREFIX/bin/load-spark-env.sh"
+. "/etc/spark/conf/load-spark-env.sh"
 
 if [ "$HOSTLIST" = "" ]; then
   if [ "$SPARK_SLAVES" = "" ]; then
diff -uprN spark-1.1.0-vanilla/sbin/spark-daemon.sh spark-1.1.0/sbin/spark-daemon.sh
--- spark-1.1.0-vanilla/sbin/spark-daemon.sh	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/sbin/spark-daemon.sh	2014-10-13 12:59:39.163519584 +0200
@@ -86,7 +86,7 @@ spark_rotate_log ()
     fi
 }
 
-. "$SPARK_PREFIX/bin/load-spark-env.sh"
+. "/etc/conf/spark/load-spark-env.sh"
 
 if [ "$SPARK_IDENT_STRING" = "" ]; then
   export SPARK_IDENT_STRING="$USER"
diff -uprN spark-1.1.0-vanilla/sbin/start-history-server.sh spark-1.1.0/sbin/start-history-server.sh
--- spark-1.1.0-vanilla/sbin/start-history-server.sh	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/sbin/start-history-server.sh	2014-10-13 12:46:53.085112600 +0200
@@ -28,7 +28,7 @@ sbin=`dirname "$0"`
 sbin=`cd "$sbin"; pwd`
 
 . "$sbin/spark-config.sh"
-. "$SPARK_PREFIX/bin/load-spark-env.sh"
+. "/etc/spark/conf/load-spark-env.sh"
 
 if [ $# != 0 ]; then
   echo "Using command line arguments for setting the log directory is deprecated. Please "
diff -uprN spark-1.1.0-vanilla/sbin/start-master.sh spark-1.1.0/sbin/start-master.sh
--- spark-1.1.0-vanilla/sbin/start-master.sh	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/sbin/start-master.sh	2014-10-13 12:47:13.551882758 +0200
@@ -39,7 +39,7 @@ done
 
 . "$sbin/spark-config.sh"
 
-. "$SPARK_PREFIX/bin/load-spark-env.sh"
+. "/etc/conf/spark/load-spark-env.sh"
 
 if [ "$SPARK_MASTER_PORT" = "" ]; then
   SPARK_MASTER_PORT=7077
diff -uprN spark-1.1.0-vanilla/sbin/start-slaves.sh spark-1.1.0/sbin/start-slaves.sh
--- spark-1.1.0-vanilla/sbin/start-slaves.sh	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/sbin/start-slaves.sh	2014-10-13 12:58:55.645007204 +0200
@@ -38,7 +38,7 @@ done
 
 . "$sbin/spark-config.sh"
 
-. "$SPARK_PREFIX/bin/load-spark-env.sh"
+. "/etc/spark/conf/load-spark-env.sh"
 
 # Find the port number for the master
 if [ "$SPARK_MASTER_PORT" = "" ]; then
diff -uprN spark-1.1.0-vanilla/sbin/stop-slaves.sh spark-1.1.0/sbin/stop-slaves.sh
--- spark-1.1.0-vanilla/sbin/stop-slaves.sh	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/sbin/stop-slaves.sh	2014-10-13 12:46:27.579399051 +0200
@@ -22,7 +22,7 @@ sbin=`cd "$sbin"; pwd`
 
 . "$sbin/spark-config.sh"
 
-. "$SPARK_PREFIX/bin/load-spark-env.sh"
+. "/etc/spark/conf/load-spark-env.sh"
 
 # do before the below calls as they exec
 if [ -e "$sbin"/../tachyon/bin/tachyon ]; then
