diff -uprN spark-1.1.0-vanilla/bin/run-example spark-1.1.0/bin/run-example
--- spark-1.1.0-vanilla/bin/run-example	2014-09-03 08:00:33.000000000 +0200
+++ spark-1.1.0/bin/run-example	2014-10-03 14:13:40.114983883 +0200
@@ -34,11 +34,7 @@ else
   exit 1
 fi
 
-if [ -f "$FWDIR/RELEASE" ]; then
-  export SPARK_EXAMPLES_JAR=`ls "$FWDIR"/lib/spark-examples-*hadoop*.jar`
-elif [ -e "$EXAMPLES_DIR"/target/scala-$SCALA_VERSION/spark-examples-*hadoop*.jar ]; then
-  export SPARK_EXAMPLES_JAR=`ls "$EXAMPLES_DIR"/target/scala-$SCALA_VERSION/spark-examples-*hadoop*.jar`
-fi
+export SPARK_EXAMPLES_JAR=`ls "$EXAMPLES_DIR"/spark-examples-*hadoop*.jar`
 
 if [[ -z $SPARK_EXAMPLES_JAR ]]; then
   echo "Failed to find Spark examples assembly in $FWDIR/lib or $FWDIR/examples/target" 1>&2
