--- apache-storm-0.9.2-incubating-vanilla/bin/storm	2014-07-29 10:32:37.390717235 +0200
+++ apache-storm-0.9.2-incubating/bin/storm	2014-07-29 10:37:31.558524314 +0200
@@ -49,9 +49,9 @@ def init_storm_env():
         os.environ[option] = value
 
 normclasspath = cygpath if sys.platform == 'cygwin' else identity
-STORM_DIR = "/".join(os.path.realpath( __file__ ).split("/")[:-2])
-USER_CONF_DIR = os.path.expanduser("~/.storm")
-CLUSTER_CONF_DIR = STORM_DIR + "/conf"
+STORM_DIR=os.getenv("STORM_DIR")
+USER_CONF_DIR=os.getenv("USER_CONF_DIR")
+CLUSTER_CONF_DIR=os.getenv("CLUSTER_CONF_DIR")
 if (not os.path.isfile(USER_CONF_DIR + "/storm.yaml")):
     USER_CONF_DIR = CLUSTER_CONF_DIR
 
