diff -uprN kafka-0.8.1.1-src-vanilla/bin/kafka-server-start.sh kafka-0.8.1.1-src/bin/kafka-server-start.sh
--- kafka-0.8.1.1-src-vanilla/bin/kafka-server-start.sh	2014-04-22 20:37:23.000000000 +0200
+++ kafka-0.8.1.1-src/bin/kafka-server-start.sh	2014-08-04 09:35:57.908337700 +0200
@@ -19,8 +19,9 @@ then
 	echo "USAGE: $0 [-daemon] server.properties"
 	exit 1
 fi
+etc_conf_dir=/etc/kafka/conf
 base_dir=$(dirname $0)
-export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$base_dir/../config/log4j.properties"
+export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$etc_conf_dir/log4j.properties"
 export KAFKA_HEAP_OPTS="-Xmx1G -Xms1G"
 
 EXTRA_ARGS="-name kafkaServer -loggc"
